projects/scannet_offline_eval/demo/README.md

```diff
@@ -0,0 +1,48 @@
+# Demo visualization
+
+
+## Installation
```

Comment:
@alexsax I think we should move this code to home_robot/utils/demoui or something, and move the documentation to docs